content/development-of-tests.md

```diff
@@ -16,10 +16,16 @@ continuous integration (CI) feedback loops. System test suites should be made av
 well as within a shared testing environment.
 
 ## Projects
-- TODO
-
-## Examples
-- TODO
+- [Static Analysis Testing](../static-analysis-and-security-testing.md)
+- [Dynamic Analysis Testing](../dynamic-analysis.md)
+- [Container Manifest Testing](../container-application-manifest.md)
+- [Threat Modelling](../threat-modelling.md)
```

Comment:
Hmm, i'm a little confused at this section. 

My impression for projects for this section were the TDD / unit testing frameworks, etc. which target more of test authoring by humans, which I think is the  intent of the section.

Reply:
As discussed in the catch up session, I will remove the static links and include more information regarding those items.